### PR TITLE
Add signup page

### PR DESCRIPTION
**Pull request by gracelungu**
#### What does this PR do?
This PR adds the login page to the platform

#### Description of Task to be completed?
~ Design the login page
~ Handle the login logic
~ Write tests

#### How should this be manually tested?
`/login`

#### Screenshots (if appropriate)
image

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "style-loader": "0.23.1",
     "terser-webpack-plugin": "1.2.3",
     "ts-pnp": "1.1.2",
+    "typeface-open-sans": "0.0.54",
```
Comment (oesukam) [inline package.json]: @gracelungu Can we use the CDN instead to reduce on the request bundle?

File: src/actions/loginActions.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import axios from 'axios';
```
Comment (oesukam) [inline src/actions/loginActions.js]: Can we create the `axios` util for customization?

File: src/index.js [hunk 3]
```diff
@@ -8,6 +8,8 @@ import * as serviceWorker from './serviceWorker';
 import store from './store';
 import Routes from './components/Routes';
 
+import 'typeface-open-sans';
```
Comment (oesukam) [inline src/index.js]: Let us use the cdn

File: src/pages/Home/Home.jsx [hunk 4]
```diff
@@ -20,15 +18,7 @@ export class Home extends Component {
...
 
-const mapStateToProps = ({ name }) => ({
-  name,
-});
-
-const mapDispatchToProps = dispatch => ({
-  changeName: name => dispatch(changeName(name)),
-});
-
 export default connect(
```
Comment (oesukam) [inline src/pages/Home/Home.jsx]: You don't need to connect if you are not using the store

File: src/pages/Login/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+import google from '../../assets/images/google.png';
+import facebook from '../../assets/images/facebook.png';
+import github from '../../assets/images/github.png';
+import girl from '../../assets/images/girl.svg';
+import backgroundImage from '../../assets/images/man.jpg';
+
+import { login, formInputChanged } from '../../actions/loginActions';
+
+export class Login extends Component {
+  form = React.createRef();
```
Comment (oesukam) [inline src/pages/Login/Login.jsx:15]: Why the reference?

**Review by oesukam — CHANGES_REQUESTED** [review]

File: src/pages/Login/Login.jsx (hunk 5 above)
Comment (gracelungu) [reply src/pages/Login/Login.jsx:15]: To use HTML5 validation.

**Pull request merged**